pyensembl/ensembl_release.py

```diff
@@ -125,6 +126,57 @@ def run_query():
             return results
         return cached_object(pickle_path, compute_fn=run_query)
 
+    def install(self):
+        """
+        Explicitely download and index any data for this release that 
+        is not yet downloaded and/or indexed.
+        """
+        self._download(force=False)
+        self._index(force=False)
+
+    def index(self):
+        """
+        Explicitely index all data for this release, regardless of
+        whether it is already indexed. Raises an error if data is
+        not downloaded.
+        """
+        self._index(force=True)
+
+    def _index(self, force):
+        if self.db.create(force=force):
+            print("Annotation data for release %s has just been indexed" %
+                  self.release)
```

Comment:
I'm less interested in seeing status update *after* some computationally intensive work than before. I would probably change this to logging.info("...")